pkg/monitor.go

```diff
@@ -169,6 +170,19 @@ func (m *Monitor) UnmarshalJSON(data []byte) error {
 			raw[f] = v
 		}
 	}
+	// We don't need full AlertContact structure, we need only its ID
```

Comment:
Well, if we're going to do this, we might as well define the proper struct!

Reply:
Proper struct defined for the Monitor (`pkg/monitor.go`). There we have 
```
type Monitor struct {
        ...
	AlertContacts []string `json:"alert_contacts,omitempty"`
        ...
}
```

Reply:
Sorry, I misunderstood your comment, then. I thought you were saying you weren't going to unmarshal the complete struct. Are you?

Reply:
Yes. We have structure for AlertContact, and we have field AlertContacts (type []string) in Monitor structure. We don't need complete AlertContacr struct in Monotor, that's why I used only AlertContact.Id. 

May be I misunderstood your first question. What did you mean by 

> Well, if we're going to do this, we might as well define the proper struct!

Reply:
@bitfield Are you still remember about this?

Reply:
I haven't forgotten about this, but it needs a non-trivial amount of time to get my head back into this project and review your code properly. I'll try to get to it as soon as I can!